data/active_adapters.py

```diff
@@ -198,6 +200,7 @@
     'mouse_variant': MouseGenomesProjectAdapter,
     'variant_disease': ClinGen,
     'variant_disease_gene': ClinGen,
+    'variant_elements': BlueSTARRVariantElement,
```

Comment:
the key here is the same as line 120, so 'variant_elements' should be 'bluestarr_variant_elements', this is also used as the adapter arg for data_parser.py (data_parser.py --adapter bluestarr_variant_elements ...)

Reply:
done